packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -58,20 +59,23 @@ const HistoryPage = () => {
   const validQueryParams = buildValidGetParams(query);
   const page = validQueryParams.page ? Number(validQueryParams.page) : 1;
 
-  const versionsResponse = useGetHistoryVersionsQuery({
-    contentType: slug!,
-    ...(documentId ? { documentId } : {}),
-    ...validQueryParams,
-  });
+  const versionsResponse = useGetHistoryVersionsQuery(
+    {
+      contentType: slug!,
+      ...(documentId ? { documentId } : {}),
+      // Omit id since it's not needed by the endpoint and caused extra refetches
+      ...omit(['id'], validQueryParams),
```

Comment:
hmm I think we should avoid lodash here according to [fe guidelines](https://contributor.strapi.io/guides/fe-coding-guidelines#a-special-note-on-lodash)